builder/amazon/common/test_helper_funcs.go

```diff
@@ -0,0 +1,25 @@
+package common
```

Comment:
now this little stub can be used anywhere instead of just in the common pkg, which made it accessible to your ebsvolume tests